gamification/data/points.py

```diff
@@ -0,0 +1,52 @@
+def get_points(activity_type):
+    """
+    Get points_dict for issue and merge_request.
+
+    This method contains the points assigned to the activity
+    variables, which will be used throughout the gamification
+    system.
+    """
+    if activity_type == 'issue':
+        points_dict = {
+            'difficulty/newcomer': 5,
+            'difficulty/low': 7,
+            'difficulty/medium': 15,
+            'difficulty/high': 20,
+            'importance/low': 5,
+            'importance/medium': 7,
+            'importance/high': 10,
+            'importance/critical': 13,
+            'area/documentation': 5,
+            'area/upstream': 7,
+            'area/genericbears': 10,
+            'area/lintbears': 7,
+            'area/nativebears': 5,
+            'type/bug': 5,
+            'type/feature': 5,
+            'bear proposal': 5,
+            'status/invalid': -5,
+            'status/duplicate': -5,
+        }
+        return points_dict
+    if activity_type == 'merge_request':
```

Comment:
elif

but again, declare these as module level CONSTANTS.

There is no benefit putting them inside a function.